mix.exs

```diff
@@ -21,7 +21,7 @@ defmodule EctoPSQLExtras.Mixfile do
     [
       {:table_rex, "~> 3.1.1"},
       {:ecto_sql, "~> 3.4"},
-      {:postgrex, "~> 0.15.7 or ~> 0.16.0"},
+      {:postgrex, "~> 0.15.7 or ~> 0.16.0 or ~> 0.17.0"},
```

Comment:
I considered changing the version to be ">=15.7", but followed the precedent in the current declaration, figuring there was a reason for it. Looking back at #41 , I'll update the declaration here as well as the GH action's 'step' names